src/interfaces/pyo3/py_abe_policy.rs

```diff
@@ -0,0 +1,168 @@
+use abe_policy::{Attribute as AttributeRust, Policy as PolicyRust, PolicyAxis as PolicyAxisRust};
+use pyo3::{exceptions::PyException, exceptions::PyTypeError, prelude::*, types::PyType};
+
+// Pyo3 doc on classes
+// https://pyo3.rs/v0.16.2/class.html
+
+/// An attribute in a policy group is characterized by the axis policy name
+/// and its unique name within this axis.
+#[pyclass]
+pub struct Attribute {
+    inner: AttributeRust,
+}
+
+#[pymethods]
+impl Attribute {
+    /// Create a Policy Attribute.
+    ///
+    /// - `axis`    : policy axis the attributes belongs to
+    /// - `name`    : unique attribute name within this axis
+    #[new]
+    pub fn new(axis: &str, name: &str) -> Self {
+        Attribute {
+            inner: AttributeRust::new(axis, name),
+        }
+    }
+
+    /// Return a string representation of the Attribute
+    #[allow(clippy::inherent_to_string)]
+    pub fn to_string(&self) -> String {
+        format!("{}", &self.inner)
+    }
+
+    #[classmethod]
+    pub fn from_string(_cls: &PyType, string: &str) -> PyResult<Self> {
+        let attribute = match AttributeRust::try_from(string) {
+            Ok(att) => att,
+            Err(e) => return Err(PyException::new_err(e.to_string())),
+        };
+        Ok(Attribute { inner: attribute })
+    }
+}
+
+/// Defines a policy axis by its name and its underlying attribute names.
+///
+/// If `hierarchical` is set to `true`, we assume a lexicographical order based
```

Comment:
We do not have any `hierarchical` variable here.

```suggestion
/// If the axis is defined as hierarchical, we assume a lexicographical order
```